src/pages/what-we-do.pug

```diff
@@ -20,30 +20,30 @@ block pageComponents
   section.section
     .container
       +block({
-        blockHeading: "Research",
-        blockContent: "<p>We use techniques like <strong>staff shadowing</strong> and <strong>behavioural analytics</strong> to help uncover automation opportunities, and recorded user sessions to validate any assumptions and establish an efficiency baseline in the existing processes.</p><p>We'll make a series of recommendations on how software could automate both internal processes and external data-capture, along with rationale on the benefits to your staff, customers, and bottom line.</p>",
+        blockHeading: "1. Analysis",
+        blockContent: "<p>First, we <strong>listen to you</strong>. Before anything else, we'll seek to <strong>understand your business</strong>, to understand <strong>the problems you're having</strong>, and to understand <strong>your goals</strong> for automation.</p><p>Taking time to listen helps us to get a clear picture of what's important to you, and how we can best work with you to <strong>deliver value</strong>.</p><p>Once we have a shared understanding of your problems and goals, we'll then use techniques like <strong>staff shadowing</strong>, <strong>interviews</strong>, <strong>data analysis</strong>, and <strong>user research</strong> to uncover automation opportunities.</p>",
         blockImage: {
           src: "/assets/images/insight.jpg",
-          alt: ""
+          alt: "Analysing your business"
         }
       })
 
       +block({
-        blockHeading: "Design",
-        blockContent: "<p>We design automated solutions to bottlenecks in your business processes, using <strong>sketching</strong>, <strong>wireframing</strong>, <strong>prototyping</strong>, and <strong>visual design</strong> to communicate a clear vision for how your software will work.</p><p>Introducing design-thinking early in the process allows us to identify and eliminate hurdles, while promoting confidence in the end result from stakeholders, staff, and customers.</p>",
+        blockHeading: "2. Recommendations",
+        blockContent: "<p>We'll put together a report detailing a series of recommendations for specific software automations that will <strong>increase the speed</strong>, <strong>accuracy</strong>, <strong>scaleability</strong>, <strong>competitiveness</strong>, and <strong>global reach</strong> of your business.</p><p>In each case, we'll provide a clear explanation of how the automation works, how it's best implemented, and the value it will deliver. This way, you can be confident of a <strong>return on investment</strong>.</p><p>If you'd like to handle the implementation yourself from this point, that's fine. <strong>We can treat our recommendations as a standalone engagement</strong>. You're free to take the recommendations and implement them on your own, should you wish.</p>",
```

Comment:
- As above, I wonder whether this could be condensed into two paragraphs.
- There are statements here, such a competitiveness (vague) and global reach (speculative) that, in my opinion, detract from the core offering.

How do you feel about this:

> <p>We'll outline automation opportunities that will <strong>increase the speed</strong>, <strong>accuracy</strong>, and <strong>scaleability</strong> of your business. Each recommendation will have a clear explanation of its value, helping you prioritise and budget based on the <strong>return on investment</strong>.</p><p>If at this point you'd like to handle the implementation yourself, either in-house or through a third-party, <strong>we can treat our recommendations as a standalone engagement</strong>. You'll never be locked into services you don't need or want.
</p>

Again, if it's good, here's the HTML:

`<p>We'll outline automation opportunities that will <strong>increase the speed</strong>, <strong>accuracy</strong>, and <strong>scaleability</strong> of your business. Each recommendation will have a clear explanation of its value, helping you prioritise and budget based on the <strong>return on investment</strong>.</p><p>If at this point you'd like to handle the implementation yourself, either in-house or through a third-party, <strong>we can treat our recommendations as a standalone engagement</strong>. You'll never be locked into services you don't need or want.</p>`